modules/gke-cluster/main.tf

```diff
@@ -88,6 +88,12 @@ resource "google_container_cluster" "cluster" {
     network_policy_config {
       disabled = ! var.enable_network_policy
     }
+
+    istio_config {
+      disabled = ! var.enable_istio
+      auth = var.istio_auth
```

Comment:
Can you run `terraform fmt`?

Reply:
sorry about this. Fixed.

Reply:
This block shouldn't have a newline underneath it either.

Reply:
Also I'm wondering if we should only add the `istio_config` block is the `enable_istio` parameter is set. I'm fairly certain we could adapter code like: https://github.com/gruntwork-io/terraform-google-gke/blob/74481db83eacfc047953bb1ec6f590e7ae93591f/modules/gke-cluster/main.tf#L47 to achieve this.